simpa/core/simulation_modules/reconstruction_module/reconstruction_utils.py

```diff
@@ -278,9 +278,6 @@ def preparing_reconstruction_and_obtaining_reconstruction_settings(
         raise AttributeError("Please specify a value for SPACING_MM in either the component_settings or"
                              "the global_settings.")
 
-    # get device specific sensor positions
```

Comment:
I think this comment rather belongs to line 284 than the deleted line 282 and thus should probably stay in the code